bin/MadGraph5_aMCatNLO/cards/production/pre2017/14TeV/DYJets_mll-4to10/DYJets_mll-4to10_proc_card.dat

```diff
@@ -0,0 +1,20 @@
+import model sm-ckm_no_b_mass
+define p = g u c d s u~ c~ d~ s~
```

Comment:
please remove the duplicate definitions of proton, jet, leptons and the unused neutrinos